message_handler.py

```diff
@@ -46,6 +48,23 @@ def handle_message_invalid():
     '''
     return 'I don\'t understand your accent. Try !help'
 
+def handle_message_crate_time():
+    # Get current UTC time (UTC is common for world).
+    current_utc_time = datetime.datetime.utcnow()
+
+    # Turn UTC time to correct time zone (-7).
+    current_timezone_time = current_utc_time + datetime.timedelta(hours=-7)
```

Comment:
Can python do timezone with daylight savings?

Reply:
Need to add pydoc type comments here